### PR TITLE
(feat) Reaction v1.13.0 compatibility

### PR DESCRIPTION
**Pull request by ajporlante**
This fixes several Reaction `v1.13.0` compatibility and styling issues of the plugin.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,32 +47,92 @@
...
           }
         }
       ]
     ],
-    "presets": [
-      "env",
-      "react",
-      "stage-2"
-    ]
+    "env": {
```
Comment (Akarshit) [inline package.json]: Will it work if we just don't have these babel configs in this package.json. Won't they be just picked up from the reaction's package.json?

**Comment by ajporlante** [comment]
The ui-tagnav is a necessary import, so I uncommented it. The babel config has to be there in `package.json` for the alias import, but I cut it shorter by removing the babel config for the devserver.

File: client/components/core/ui/index.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
 import "./icon/index";
-import "./tags";
+// import "./tags";
```
Comment (Akarshit) [inline client/components/core/ui/index.js]: @ajporlante we can remove this line too.

**Comment by ajporlante** [comment]
Uncommented the tags import. Missed it! Thanks @Akarshit !

File: client/components/product-variant/customer/productGridItem.js [hunk 3]
```diff
@@ -1,22 +1,39 @@
...
-    showFeaturedLabel: true
+    showFeaturedLabel: true,
+    similarProducts: false
   };
 
+  handleClick = (event) => {
+    event.preventDefault();
+    Router.go(this.productURL);
+    if (this.props.similarProducts) {
+      $("html,body").animate(
```
Comment (zenweasel) [inline client/components/product-variant/customer/productGridItem.js]: I think adding jQuery into React is an anti-pattern. Is there no way to do this animation with one of the React libraries we already include?

**Comment by zenweasel** [comment]
I had one note, but I want to just get this merged so it's working for everybody. We can update separately.

**Pull request merged**